py/desidatamodel/render.py

```diff
@@ -0,0 +1,63 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+====================
+desidatamodel.render
+====================
```

Comment:
consider giving this file a more specific name, since it is rendering the very specific case of the column_descriptions.csv file, not rendering the desidatamodel as a whole as the module name might imply.

Reply:
Will do.